tests/libTestUtils/TestUtils.h

```diff
@@ -45,15 +46,26 @@ uint8_t Dist1to99();
 uint8_t DistUint8();
 uint16_t DistUint16();
 uint32_t DistUint32();
+uint32_t DistUint64();
+uint32_t DistUint128();
```

Comment:
This seems unimplemented.

Reply:
Copy/paste issue. 
uint32_t DistUint128() - removed, I wasn't able to find numerical limit for 128bit integer